src-docs/spec/sendUpdate.md

```diff
@@ -35,6 +15,52 @@ In these cases, you can still call `sendUpdate()`,
 however, the update won't be sent to other peers
 and you won't get the update by [`setUpdateListener()`](./setUpdateListener.html).
 
+The `update` object has the following properties:  
+
+- `update.payload`: string, number, boolean, array, object or `null`.
+   MUST NOT be `undefined`.
+   Everything that is not JSON serializable will be skipped,
+   this especially affects Binary data buffers as used in `File`, `Blob`, `Int*Array` etc.;
+   if needed, use eg. base64.
+
+- `update.info`: optional, short, informational message that will be added to the chat,
+   e.g. "Alice voted" or "Bob scored 123 in MyGame".
+   Do not add linebreaks; implementations will truncate the text at about 50 characters or less.
```

Comment:
btw "50 character or less" for info message sounds super restrictive, some users's display name alone (ex. name and lastname) will take up most of that